### ISSUE TITLE
Clean up tutorial

### ISSUE THREAD

**Issue by andregmoeller** [go-swagger/go-swagger#275]
I followed the todo list tutorial and feel that there is room for improvement:

- the copyright year needs to get updated
- I like YAML, but I don't see YAML files often. I was a little confused about the swagger.yml as in some
lines I saw strings in double quotes besides strings enclosed by single quotes and some without any quotes at all. Furthermore, I feel that the usage of flow style is a little confusing.
- The completed spec includes a "- https", which makes it more difficult to follow the tutorial because you have to specify startup parameters like `tis-certificate` and `tls-key`.
- There are some differences between the tutorial and the example source code:
  - `var store = make(map[int64]models.Item)` <=> `var items = make(map[int64]*models.Item)`
  - `var ids int64` <=> `var lastID int64`
  - `curl -i localhost:8765` returns `nil` instead of `[]`: in order to fix it you have to add `result = make([]*models.Item, 0)` to the `allItems` function (see [configure_todo_list.go](https://github.com/go-swagger/go-swagger/blob/master/examples/tutorials/todo-list/server-complete/cmd/todo-list-server/configure_todo_list.go#L74-L84))
- the `allItems` function doesn't interpret the `since` parameter the way I expected it. In my opinion [line 79](https://github.com/go-swagger/go-swagger/blob/master/examples/tutorials/todo-list/server-complete/cmd/todo-list-server/configure_todo_list.go#L79) has to be changed to `if since == 0 || since <= id {`
- the [`FindTodosHandlerFunc`](https://github.com/go-swagger/go-swagger/blob/master/examples/tutorials/todo-list/server-complete/cmd/todo-list-server/configure_todo_list.go#L108-L116) ignores the `Since`and `Limit`parameters. I feel that it should be changed to something like
```
	api.TodosFindTodosHandler = todos.FindTodosHandlerFunc(func(params todos.FindTodosParams) middleware.Responder {
		mergedParams := todos.NewFindTodosParams()
		mergedParams.Since = swag.Int64(0)
		if params.Since != nil {
			mergedParams.Since = params.Since
		}
		if params.Limit != nil {
			mergedParams.Limit = params.Limit
		}
		return todos.NewFindTodosOK().WithPayload(allItems(*mergedParams.Since, *mergedParams.Limit))
	})
```

To sum it up, I would like to "clean up" the tutorial and example source code a bit.

**Comment by casualjim** [comment]
agree on all points except the alItems one 
The since is for all ids since the one provided (so if since is 1 then we want id's 2,3,4 and so on)

wrt to https, I think it's still useful to include it in the tutorial but perhaps it should be an extra step and some more explanation about what and why?

> To sum it up, I would like to "clean up" the tutorial and example source code a bit.

Does that mean a PR is coming?

**Comment by andregmoeller** [comment]
> agree on all points except the alItems one 
> The since is for all ids since the one provided (so if since is 1 then we want id's 2,3,4 and so on)

Right now, the program code looks like this:
```
if since == 0 || since > id {
	result = append(result, item)
}
```
So, if since is 3 you would get the ids 1 and 2. You wouldn't get 4, 5, 6 and so on.

Probably the program code should look like this:
```
if since == 0 || id > since {
	result = append(result, item)
}
```

> Does that mean a PR is coming?

Yes!
